api/v1/version_service.proto

```diff
@@ -19,41 +19,47 @@ package api.v1;
 
 import "validate/validate.proto";
 
-option go_package = "github.com/koor-tech/version-service/api/v1";
+option go_package = "github.com/koor-tech/version-service/api/v1;apiv1";
 
 service VersionService {
   rpc Operator(OperatorRequest) returns (OperatorResponse) {}
 }
 
-message CurrentVersions {
-  string kube = 1;
-  string koor_operator = 2;
-  string ksd = 3;
-  string ceph = 4;
+// ProductVersions describes a map of products with version strings
+message ProductVersions {
+  string kube = 1 [(validate.rules).string.pattern = "^v?\\d+(?:\\.\\d+){0,2}(?:\\-\\d+)?$"];
+  string koor_operator = 2 [(validate.rules).string.pattern = "^v?\\d+(?:\\.\\d+){0,2}(?:\\-\\d+)?$"];
+  string ksd = 3 [(validate.rules).string.pattern = "^v?\\d+(?:\\.\\d+){0,2}(?:\\-\\d+)?$"];
+  string ceph = 4 [(validate.rules).string.pattern = "^v?\\d+(?:\\.\\d+){0,2}(?:\\-\\d+)?$"];
```

Comment:
These versions might be too strict, depending on how the version numbers are retrieved on the operator side:

```
$ ceph version
ceph version 17.2.6 (d7ff0d10654d2280e08f1ab989c7cdf3064446a5) quincy (stable)
```

Reply:
I'm using the version number [in the tag](https://quay.io/repository/ceph/ceph?tab=tags&tag=latest) of the container name, which is either:

- v17
- v17.2
- v17.2.6
- v17.2.6-20230712

All of which [match the regex](https://regex101.com/r/8Dstkz/1).

Reply:
Then we gotta make sure we never deploy tags that aren't semver numbers :D